### ISSUE TITLE
Add into Transcriber Help and/or FAQ what to do if you haven't heard from your coordinator

### ISSUE THREAD

**Issue by DeniseColbert** [FreeUKGen/MyopicVicar#1509]

**Comment by Captainkirkdawson** [comment]
The role should be used NOT the person

**Comment by SteveBiggs** [comment]
I agree Kirk, but it must be easy for the transcriber to find the contact details of who is currently in the role.

**Comment by stsccfr** [comment]
Transcribers have an action button called Coordinators List which lists contact info for all county and syndicate coords.  We could change the label on this button to something like FreeREG Contacts and add in the contact info for the head of FreeREG and the head of FUKG.

The little screenshot image app/assets_freereg/images/transcriber-actions.png needs to be updated to show the Coordinators list button, but would also need to be updated anyway if we change the label on that button.

**Comment by stsccfr** [comment]
Closing in favour of #2909
